operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/impl/AbstractSystemConfigurationImpl.java

```diff
@@ -352,4 +352,9 @@ public String getOperationOutcomeProvider(String provider) {
     public boolean hasStorageProvider(String storageProvider) {
         return FHIRConfigHelper.getStringProperty("fhirServer/bulkdata/storageProviders/" + storageProvider + "/type", null) != null;
     }
+
+    @Override
+    public boolean enableSkippableUpdates() {
+        return FHIRConfigHelper.getBooleanProperty("fhirServer/bulkdata/core/enableSkippableUpdates", Boolean.TRUE);
```

Comment:
Do you think it's worth making`fhirServer/bulkdata` prefix a constant? Sometimes I find the resulting concatenation harder to read...so not sure.

Reply:
it's subtle, but it's a big change in that class.  if we change this one, we change them all in the class. I'm not sure if it is a benefit in this case.